packages/spektate/src/repository/IGitHub.ts

```diff
@@ -107,13 +107,13 @@ export const getPullRequest = (
           id: pr.number,
           mergedBy: pr.merged_by
             ? {
-                imageUrl: pr.merged_by.avatar_url
-                  ? pr.merged_by.avatar_url
-                  : "",
-                name: pr.merged_by.login ? pr.merged_by.login : "",
-                url: pr.merged_by.url ? pr.merged_by.html_url : "",
-                username: pr.merged_by.login ? pr.merged_by.login : ""
-              }
+              imageUrl: pr.merged_by.avatar_url
```

Comment:
Same for this block 
<img width="635" alt="image" src="https://user-images.githubusercontent.com/3119513/79785249-5fdc6100-82f8-11ea-8052-1cf37efeeb19.png">